testsuite/metamodelica/meta/llvm/Makefile

```diff
@@ -0,0 +1,66 @@
+TEST = ../../../rtest -v
```

Comment:
@perost  @sjoelund @adrpo 
Do you guys know if there is a good alternative for writing my own set of mos scripts 
Is there any other standardised way of passing specific flags using rtest/partest?